CRM/PCP/Page/PCPInfo.php

```diff
@@ -185,10 +185,11 @@ public function run() {
     if (!empty($entityFile)) {
       $fileInfo = reset($entityFile);
       $fileId = $fileInfo['fileID'];
+      $altText = $fileInfo['description'] ?? '';
```

Comment:
Because `'description'` is not automatically html-escaped (per https://github.com/civicrm/civicrm-core/blob/5.43/CRM/Utils/API/HTMLInputCoder.php#L55) it needs to be escaped here before inserting it into html.

Reply:
Good catch. I've updated this to use `htmlspecialchars`, but to be honest I wasn't sure if CiviCRM has a better function to be using here - let me know if so!

Reply:
That works. Thanks.